android/app/src/main/java/com/msupplymobile/MainActivity.java

```diff
@@ -1,18 +1,8 @@
 package com.msupplymobile;
 
 import com.facebook.react.ReactActivity;
-import com.babisoft.ReactNativeLocalization.ReactNativeLocalizationPackage;
-import io.realm.react.RealmReactPackage;
-import com.oblador.vectoricons.VectorIconsPackage;
-import com.bugsnag.BugsnagReactNative;
-import android.os.Bundle;
 
 public class MainActivity extends ReactActivity {
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-      super.onCreate(savedInstanceState);
-      BugsnagReactNative.start(this);
```

Comment:
RIP bugsnag

Reply:
Sorry about this, missed putting it back.